recipes-devtools/rust/rust.inc

```diff
@@ -78,10 +78,11 @@ def llvm_features_from_tune(d):
     if 'dsp' in feat:
         f.append("+dsp")
 
-    if d.getVar('ARM_THUMB_OPT') is "thumb":
-        if not feat.isdisjoint(v7):
-            f.append("+thumb2")
-        f.append("+thumb-mode")
+    if 'thumb' in feat:
+        if d.getVar('ARM_THUMB_OPT') is "thumb":
```

Comment:
Isn't just checking for the feature enough? Or why is this check of `ARM_THUMB_OPT` needed too?

Reply:
Since there's not much documentation that I can find on the variables involved other than ```TUNE_FEATURES``` I'm not actually 100% certain. I think though that based on the code it's possible that ```TUNE_FEATURES``` will include thumb but ```ARM_INSTRUCTION_SET``` will be arm, in which case ```ARM_THUMB_OPT``` will be arm. I'm assuming that if someone sets ```ARM_INSTRUCTION_SET``` to arm that they actually want it to be compiled as arm even though thumb is in the ```TUNE_FEATURES```.